### PR TITLE
Added GUILD_SCHEDULED_EVENT_* constants.

### PR DESCRIPTION
**Pull request by shehi**
These constants are critical for our implementation. Please kindly merge this as soon as possible with a release.

P.S.: Thank you for this library! You can't imagine how life-saving this was for us! Please keep up the good work. We will be donating to this project once our implementation is live, rest assured!

### CODE REVIEW

File: src/Discord/WebSockets/Event.php [hunk 1]
```diff
@@ -65,13 +69,9 @@ abstract class Event
     public const CHANNEL_UPDATE = 'CHANNEL_UPDATE';
     public const CHANNEL_PINS_UPDATE = 'CHANNEL_PINS_UPDATE';
 
-    // Threads
```
Comment (davidcole1340) [inline src/Discord/WebSockets/Event.php:68]: Were these removed by accident?

Comment (shehi) [reply src/Discord/WebSockets/Event.php:68]: Dammit, might've been sleeping behind keyboard. Fixing now...

**Comment by davidcole1340** [comment]
Thanks!

**Review by shehi — COMMENTED** [review]
Good to go.

**Comment by shehi** [comment]
Closing remarks: you should set up project Sponsoring in here. https://github.com/electron-userland/electron-builder does this greatly, where open-source users like me sponsor the changes they want, and devs work on those sponsored requests, thus earning money for their time. Good luck!

**Comment by davidcole1340** [comment]
Thank you 😄 I don't really have the time to accept money for sponsored requests, but if you do have a request feel free to create an issue.

**Pull request merged**